include/vcpkg/installedpaths.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <vcpkg/base/files.h>
+
+#include <vcpkg/binaryparagraph.h>
+#include <vcpkg/packagespec.h>
+#include <vcpkg/triplet.h>
+
+namespace vcpkg
+{
+    struct InstalledPaths
+    {
+        explicit InstalledPaths(Path&& root) : m_root(std::move(root)) { }
```

Comment:
I still think the default should be to accept by `const&` and only add `&&` forms where perf suggests it matters, but not a big deal considering the only caller passes a temporary.